filter/ArticleCrossrefXmlFilter.php

```diff
@@ -270,6 +270,13 @@ public function createJournalArticleNode($doc, $submission)
             $journalArticleNode->appendChild($licenseNode);
         }
 
+        // version info
```

Comment:
Maybe to leave it out for now, for us to have time to work it fully out -- for 3.6 we are working on some versioning, DOIs for versioning, version numbers/names...